packages/node_modules/@webex/common/src/patterns.js

```diff
@@ -11,11 +11,22 @@
  */
 export default {
   /**
-   * Matches an email address by requiring an @ and excluding spaces
-   * @todo add a better email address matcher
+   * Used to validate a string is explicitly an email and allows for immediate
+   * validation of emails provided by services, such as conversations. See
+   * [this link]{@link https://ihateregex.io/expr/email-2} for additional info.
+   *
    * @type {RegExp}
    */
-  email: /^[^\s]+?@[^\s]+?$/,
+  email: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
Improved this regex. Note, it functions very differently in contrast to the `containsEmails` regex.
This one:
- Validates a string is an email.
- Is consumed by conversation and user plugins [primarily]
  - Used for validating and organizing conversation activities
  - Used for validating user details in the user plugin